README.md

```diff
@@ -199,6 +199,48 @@ of `distutils` (but not `sysconfig`)](https://ffy00.github.io/blog/02-python-deb
 While we always recommend the use of virtual environments, uv considers them to be required in
 these non-standard environments.
 
+### Persistent configuration
+
+uv supports persistent configuration at both the project- and user-level.
+
+Specifically, uv will search for a `pyproject.toml` or `uv.toml` file in the current directory, or
+in the nearest parent directory.
+
+If a `pyproject.toml` file is found, uv will read configuration from the `[tool.uv.pip]` table. For
```

Comment:
Should we talk about searching _past_ these if they don't contain a `tool.uv.pip` section?